### ISSUE TITLE
Running `config` should not produce an error

### ISSUE THREAD

**Issue by alondahari** [nobe4/gh-not#20]
It should probably create an empty (or default) config file for you if it doesn't exist

**Comment by alondahari** [comment]
same goes for the cache file

**Comment by nobe4** [comment]
I believe this has been fixed with #19:

```
$ go run cmd/gh-not/main.go config --config ./missing.yaml --verbosity 5
time=2024-06-11T11:57:45.335+02:00 level=WARN source=/Users/nobe4/dev/nobe4/gh-not/internal/config/config.go:34 msg="config file not found, using default configuration" path=./missing.yaml
time=2024-06-11T11:57:45.336+02:00 level=DEBUG source=/Users/nobe4/dev/nobe4/gh-not/internal/cmd/root.go:77 msg=config config="&{Cache:{TTLInHours:168 Path:/Users/nobe4/.local/state/gh-not/cache.json} Rules:[]}"
./missing.yaml
cache:
    ttl_in_hours: 168
    path: /Users/nobe4/.local/state/gh-not/cache.json
rules: []
```

I don't think `gh not config` should create an empty config by default, but I like the idea of creating a `init` command to have a starting point.
